### ISSUE TITLE
History of distal radius fracture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10720]
**Preferred term label:**
History of distal radius fracture

**Synonyms**

**Definition (free text, please give PubMed ID)**
Recent or remote history of distal radius fracture of the symptomatic arm

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Scapholunate advanced collapse (SLAC)
Distal radioulnar joint (DRUJ) arthritis
Scapholunate (SL) ligament injury

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001250
- label: History of distal radius fracture
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: 
- def: Recent or remote history of distal radius fracture of the symptomatic arm.
- comment: This finding refers to a medical history of distal radius fracture in the arm that is the focus of current medical attention. This finding may relate to scapholunate advanced collapse, distal radioulnar joint arthritis, and scapholunate ligament injury.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10720
- synonyms:
